code/iexfeed/iex.q

```diff
@@ -0,0 +1,68 @@
+\d .iex
+
+.iex.main_url: "https://api.iextrading.com";
+.iex.convert_epoch:{"p"$1970.01.01D+1000000j*x};
+.iex.reqtype:`trade
+.iex.syms:`CAT`DOG
+.iex.callback:string @[value;".u.upd";{[x;y]x set {[t;x]x}}".u.upd"] 
+.iex.callbackhandle:0
+.iex.callbackconnection:`
+.iex.init:{[x]
+   if[`main_url in key[x];.iex.main_url:x[`main_url]];
+   if[`prefix in key[x];.iex.prefix:x[`prefix]];
+   if[`quote_suffix in key[x];.iex.quote_suffix:x[`quote_suffix]];
+   if[`trade_suffix in key[x];.iex.trade_suffix:x[`trade_suffix]];
+   if[`syms in key[x];.iex.syms: upper x[`syms]];
+   if[`reqtype in key[x];.iex.reqtype:x[`reqtype]];
+   if[`callbackconnection in key[x];.iex.callbackhandle :neg[hopen[.iex.callbackconnection:x[`callbackconnection]]]];
+   if[`callbackhandle in key[x];.iex.callbackhandle:x[`callbackhandle]];
+   if[`callback in key[x];.iex.callback: $[.iex.callbackhandle=0; string @[value;x[`callback];{[x;y]x set {[t;x]x}}[x[`callback]]]; x[`callback] ]  ];
+   if[`upd in key[x]; .iex.upd:x[`upd]];
```

Comment:
I would replace all mentions of `key[x];` with `key x;`. I also think @jonathonmcmurray highlighted that this whole function could be better served with [`.Q.def`](http://code.kx.com/q/ref/dotz/#zx-argv).

Another alternative for each `if` statement that sets a value with no other action:
``.iex,:inter[`main_url`prefix;key x]#x``